src/cppcache/integration-test/testThinClientExecuteFunctionPrSHOP.cpp

```diff
@@ -563,7 +563,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, Client1OpTest2)
       LOGINFO("FETimeOut begin onRegion");
       ExecutionPtr RexecutionPtr = FunctionService::onRegion(regPtr0);
       CacheableVectorPtr fe =
-          RexecutionPtr->withArgs(CacheableInt32::create(5000))
+          RexecutionPtr->withArgs(CacheableInt32::create(5000 * 1000))
```

Comment:
Yes. Ultimately we should consider removing the continuity issue between the client API and the server. The client is taking the timeout as seconds but sending to the server as milliseconds. Ultimately taking a chrono::duration would make the API more clear. The test would still need to be very explicit on the use of milliseconds are the argument to the function.